packages/pn-personagiuridica-webapp/src/component/Deleghe/DelegationsElements.tsx

```diff
@@ -67,16 +151,58 @@ export const Menu: React.FC<{
     ];
   };
 
+  const subtitleModal =
+    menuType === 'delegates'
+      ? t('deleghe.subtitle_revocation', { recipient: userLogged?.name })
+      : t('deleghe.subtitle_rejection', { delegator: name });
+
   return (
     <>
+      <ConfirmationModal
+        open={showConfirmationModal}
+        title={
+          menuType === 'delegates'
+            ? t('deleghe.revocation_question', { delegator: name })
+            : t('deleghe.rejection_question', { delegator: name })
+        }
+        subtitle={subtitleModal}
+        onConfirm={handleConfirmClick}
+        onConfirmLabel={
+          menuType === 'delegates'
+            ? t('deleghe.confirm_revocation')
+            : t('deleghe.confirm_rejection')
+        }
+        onClose={onCloseModal}
+        onCloseLabel={t('button.annulla', { ns: 'common' })}
+      />
+
+      {verificationCode && (
```

Comment:
Add the condition menuType === 'delegates'. So verificationCode && menuType === 'delegates' && ...

Reply:
done